### DISCUSSION TITLE
How to delay ScrollSeekConfiguration exit to wait 1 second before returning true?

### DISCUSSION THREAD

**Discussion by wiktoriavh** [petyosi/react-virtuoso#593] [Q&A]
Here is a gif of me scrolling through the table, and for me, that is fast scrolling. Since I stop for a bit to move my finger up and scroll again, the items are rendering. I would like to have a  delay before it returns true. I tried to create a debounce function, which in itself worked when I tried it with a simple console.log, but when I tried to implement it into the `exit` config, it doesn't work. It doesn't even wait at all and will just return false and not update.

Here is the sandbox with my code: https://codesandbox.io/s/elegant-ptolemy-yiy0d1?file=/src/App.js

So the question is: How do I delay exit?

![lmao](https://user-images.githubusercontent.com/62669899/155705880-202ac408-dcc8-49c3-b265-c110c1ec1884.gif)

**Comment by petyosi** [comment]
You can't do that at the moment. A delay like the one you're thinking of can be implemented by [changing the logic here](https://github.com/petyosi/react-virtuoso/blob/master/src/scrollSeekSystem.ts). I'm open to PRs on the matter.

**Reply by wiktoriavh** [reply]
Hmm. That's a bummer. I will check that out. Where do I find the repo of `@virtuoso.dev/urx` ? I want to see what `u` does.

**Reply by petyosi** [reply]
Here are the docs: https://urx.virtuoso.dev/
And here's the repo: https://github.com/virtuoso-dev/urx

**Reply by wiktoriavh** [reply]
I tried to make it run locally but had no luck.  
I clone the repo with the correct EOL for me (which is CRLF). Then I installed the package into another project, so I can test it visually. No errors or warnings, but the component just didn't render at all. At this point, I don't know if I did anything wrong or if it's something else.

package.json of the project where I installed the cloned repo.
```json
{
    "react-virtuoso": "D:\\0--Projekte\\react-virtuoso",
}
```
And the location of the project is a sibling of react-virtuoso.
`D:\0--Projekte\react-virtuoso`
`D:\0--Projekte\virtualise-list-test`  
![image](https://user-images.githubusercontent.com/62669899/155965354-64808e42-59ac-4483-9d80-f6f3ef2c2684.png)  
It's inside my node_modules
![image](https://user-images.githubusercontent.com/62669899/155965381-5bd5060b-bdd0-4e3e-bd9a-884a2ffd549d.png)  
and inside my yarn.lock
![image](https://user-images.githubusercontent.com/62669899/155965394-e7762229-91fd-485f-a107-4a8cce07d138.png)

I used CRA to start the project. But when I ran `yarn start` to see it in the browser, it was just empty. Nothing was rendered unless I would remove `TableVirtuoso` from the return statement.

Maybe you can give me some hints on what I did wrong or what I should do. I wanted to work on the package and add the debounce, but can't do it without being able to run and test it as well.

**Reply by petyosi** [reply]
You don't need to go through that. Just use the `examples` directory ([check this example](https://github.com/petyosi/react-virtuoso/blob/master/examples/scroll-seek-placeholder.tsx)), like it is described in [the contribution section](https://github.com/petyosi/react-virtuoso#fixes-and-new-features).

**Reply by wiktoriavh** [reply]
Must have totally missed that. Thanks for letting me know.

**Reply by wiktoriavh** [reply]
I am assuming you're working on either Linux or Mac.
I just tried running the cloned repo, but it just fails on me. `deasync/build` is empty and cannot find bindings. In some other thread I read about `deasync` being OS specific, and I'm on windows.

I am writing here, because perhaps you got an idea on what is going on.

**Reply by petyosi** [reply]
No idea what you've encountered, sorry. You can probably use something like WSL.
